library/Cube/Ido/IdoHostStatusCubeRenderer.php

```diff
@@ -11,6 +12,19 @@
  */
 class IdoHostStatusCubeRenderer extends CubeRenderer
 {
+    protected $factsPrefix;
+
+    public function __construct(Cube $cube)
+    {
+        parent::__construct($cube);
+
+        if ($cube instanceof IdoHostStatusCube) {
+            $this->factsPrefix = 'hosts';
+        } else {
+            $this->factsPrefix = 'services';
+        }
+    }
+
```

Comment:
This is the renderer for hosts and only for hosts. There's no need to differentiate between hosts and services therefore. Please change this so you don't need the `factsPrefix` attribute.